src/notifications/use-push-notifications.tsx

```diff
@@ -110,13 +107,13 @@ export const NotificationContextProvider: React.FC = ({children}) => {
   const requestPermissions = useCallback(async () => {
     setStatus('updating');
     const permissionStatus = await requestUserPermission();
-    const token = await register(permissionStatus === 'granted');
-    if (!token) {
+    if (!fcmToken) {
       setStatus('error');
```

Comment:
When the status is error we show an error message saying "An error ocurred. Please try again later 🤞". If try again here means trying to enable to token toggle again, it will never work as `fcmToken` will always be `undefined`.

Reply:
Fixed.